main.js

```diff
@@ -0,0 +1 @@
+import './style.css'
```

Comment:
**issue**: The import statement for './style.css' is present, but the CSS file does not seem to be utilized in the HTML. Consider linking the CSS file in the HTML to apply styles.